src/app/contexts/audio/init.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+import { type Track } from "@prisma/client";
+import { createContext } from "react";
+
+export const defaultTrackData: Track = {
+  id: "XXXXXXX",
+  albumn: null,
+  duration: 300,
+  name: "Gimme the Night",
+  posterUrl: null,
+  ref: "",
+  singer: "George Benson",
+  url: "",
+  userId: "",
+};
+
+type AudioContextType = {
+  setCurrentTrack: (track: Track) => void;
+  setPlaylist: (playlist: Array<Track>) => void;
+  currentTrack: Track;
+  playlist: Array<Track>;
+};
+
+export const AudioContext = createContext<AudioContextType>({
+  setCurrentTrack: () => undefined,
+  setPlaylist: () => [],
```

Comment:
Your function should return `void`

```suggestion
  setCurrentTrack: () => {},
  setPlaylist: () => {},
```